go.sum

```diff
@@ -1788,6 +1788,7 @@ github.com/openshift/hashicorp-terraform v0.12.20-openshift-2/go.mod h1:R/dUWEZV
 github.com/openshift/hashicorp-terraform-plugin-sdk v1.6.0-openshift h1:lfMsjeRYTWs2FmLo9ak0fqV7ekzpKUtV2JkLKdlx3Zk=
 github.com/openshift/hashicorp-terraform-plugin-sdk v1.6.0-openshift/go.mod h1:H5QLx/uhwfxBZ59Bc5SqT19M4i+fYt7LZjHTpbLZiAg=
 github.com/openshift/imagebuilder v1.1.0/go.mod h1:9aJRczxCH0mvT6XQ+5STAQaPWz7OsWcU5/mRkt8IWeo=
+github.com/openshift/installer v0.16.1/go.mod h1:VWGgpJgF8DGCKQjbccnigglhZnHtRLCZ6cxqkXN4Ck0=
```

Comment:
hmm.. this is odd, why do we have the our own repo in the go.sum?

Reply:
This suggests we might have a circular dependency. One of the other libraries we are importing here uses the installer. I don't see the culprit though so maybe I'm wrong.